repository/repository.go

```diff
@@ -253,6 +255,7 @@ func (r *Repository) Store() storage.Store {
 }
 
 func (r *Repository) StorageSize() int64 {
+	fmt.Println("StorageSize()", r.storageSize, r.storageSizeDirty)
 	if r.storageSizeDirty {
```

Comment:
minus this OK; good catch, i missed this during review, sorry.

Reply:
removed